tests/unit/neuralnets/NeuralLayerTestFixture.h

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2014, Shogun Toolbox Foundation
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+
+ * 1. Redistributions of source code must retain the above copyright notice,
+ * this list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ * this list of conditions and the following disclaimer in the documentation
+ * and/or other materials provided with the distribution.
+ *
+ * 3. Neither the name of the copyright holder nor the names of its
+ * contributors may be used to endorse or promote products derived from this
```

Comment:
we should use the short style license here that is used in most places in shogun

Reply:
This one:
```
/*
 * This software is distributed under BSD 3-clause license (see LICENSE file).
 *
 * Authors: ...
 */
```